examples/revelapp/README.md

```diff
@@ -1,43 +1,23 @@
-# Welcome to Revel
+# Example Revel application
 
-A high-productivity web framework for the [Go language](http://www.golang.org/).
+This package contains an example Revel application, with Bugsnag configured.
 
+The key files for integrating Bugsnag are:
 
-### Start the web server:
+1. `app/init.go` - Sets up the Bugsnag filter.
+1. `app/controllers/app.go` - notifies about an handled error and an unhandled panic
+1. `conf/app.conf` - configures Bugsnag, in particular the API key
 
-   revel run myapp
-
-### Go to http://localhost:9000/ and you'll see:
-
-    "It works"
-
-## Code Layout
-
-The directory structure of a generated Revel application:
-
-    conf/             Configuration directory
-        app.conf      Main app configuration file
-        routes        Routes definition file
-
-    app/              App sources
-        init.go       Interceptor registration
-        controllers/  App controllers go here
-        views/        Templates directory
-
-    messages/         Message files
-
-    public/           Public static assets
-        css/          CSS files
-        js/           Javascript files
-        images/       Image files
-
-    tests/            Test suites
-
-
-## Help
-
-* The [Getting Started with Revel](http://revel.github.io/tutorial/gettingstarted.html).
-* The [Revel guides](http://revel.github.io/manual/index.html).
-* The [Revel sample apps](http://revel.github.io/examples/index.html).
-* The [API documentation](https://godoc.org/github.com/revel/revel).
+## Run the example
 
+1. Change the API key in `conf.app` to a project you've created in [Bugsnag](https://app.bugsnag.com).
+1. Inside `bugsnag-go/examples/revelapp` do:
+    ```bash
+    revel run
+    ```
+1. The application is now running. You can now visit
+    ```
+    http://localhost:9001/unhandled - to trigger an unhandled panic
+    http://localhost:9001/handled   - to trigger a handled error
+    ```
+1. You should now see events for these exceptions in your [Bugsnag dashboard](https://app.bugsnag.com).
```

Comment:
`app.conf` currently points the endpoints towards localhost, so the example app doesn't send errors to the dashboard